js/battledata.js

```diff
@@ -811,17 +811,17 @@ var Tools = {
 		return type;
 	},
 
-	loadedSpriteData: 'xy',
+	loadedSpriteData: {'xy':1, 'bw':0},
 	loadSpriteData: function(gen) {
-		if (this.loadedSpriteData === gen) return;
-		this.loadedSpriteData = gen;
+		if (this.loadedSpriteData[gen]) return;
+		this.loadedSpriteData[gen] = 1;
 
 		var path = $('script[src*="pokedex-mini.js"]').attr('src');
 		var qs = path.split('?')[1] || '';
```

Comment:
`('?' + path.split('?')[1] || '')` is equivalent to `'?' + path.split('?')[1]` because `'?' + path.split('?')[1]` is never falsy.

On the other hand, `'?' + (path.split('?')[1] || '')` is fine.

And since `qs` is now guaranteed to be a string, we can just use `qs` instead of `(qs ? qs : '')`.